Update README.md (#17)
Paperspace/gradient-cli: README.md

    and then add `. ~/paperspace_complete.sh` to your `.bashrc`, `.zshrc` or `activate` script.

    More: https://click.palletsprojects.com/en/7.x/bashcomplete/
4. Set your api key by executing the following:

    `gradient apiKey <your-api-key-here>`

   Note: your api key is cached in ~/.paperspace/config.json

   You can remove your cached api key by executing:

    `gradient logout`

5. Use gradient to run a python script remotely:

>     `gradient run myscript.py --name my_new_job`

Reviewer: ```suggestion
    `gradient run myscript.py --name <your-new-job-name>`
```

   The script will be run on the Paperspace job cluster node, and its output will be logged locally.

>    Note: this will zip and send your entire current working directory to our server if `--worskpace` was not provided

Reviewer: ```suggestion
   Note: this will zip and send your entire current working directory to our server if `--workspace` was not provided
```

>    Note: this will zip and send your entire current working directory to our server if `--workspace` was not provided

Reviewer: ```suggestion
   Note: this will archive and upload your entire current working directory to our server if `--workspace` was not provided
```

gradient experiments run [type] [--options]
```

For a full list of available commands run `gradient experiments --help`. 
Note that some options are required to create new experiment.

Running a python script by name
===============================

> You can run an python script on Paperspace server from the command line as follows:

Reviewer: ```suggestion
You can run a Python script on a Paperspace server from the command line as follows:
```

    gradient run myscript.py --name my_new_job

You can also provide additional jobs options on the command line:

    gradient run myscript.py --name my_new_job --project myproject --machineType P5000 \
     --container paperspace/tensorflow-python`

> See the [Gradient docs](https://docs.paperspace.com/gradient/jobs/about) documentation for the full list of jobs create options that can be specified.

Reviewer: ```suggestion
See the [Gradient docs](https://docs.paperspace.com/gradient/jobs/about) for the full list of jobs create options that can be specified.
```

===========
The `--workspace` option allows you to specify a workspace file or directory to upload, or a git repo link to download and merge with the container.  For example, to upload the current directory along with a script file run:

    gradient run myscript.py --name my_new_job --workspace workspace-dir

Note: `--workspace` defaults to `.`

See the Paperspae API [jobs create](https://paperspace.github.io/paperspace-node/jobs.html#.create) documentation for more details on the `--workspace` option and related options.

The `--ignoreFiles "<file-or-dir>,..."` option can be used specify a simple comma separated list of files and directories to ignore for the workspace upload:

    gradient run myscript.py --name my_new_job --workspace . --ignoreFiles "hello.py,paperspace"

The following files and directories are ignored by default: `.git`, `.gitignore`, `__pycache__`.

> Other `jobs create` options can be specified, such as `--machineType <machine type>`, `--container <container image reference>`, and `--project <project name>`.

Reviewer: ```suggestion
Other `jobs create` options can be specified, such as `--machineType <machine-type>`, `--container <container-image-reference>`, and `--project <project-name>`.
```

   Note: the above methods take precedence over use of the cached api key in
   `~/.paperspace/config.json`

3. Set the key in the `~/.paperspace/config.json` file from the command line by running:

> 

Reviewer: i believe the below needs to be "apiKey" - i believe it's case-sensitive now
